### PR TITLE
Docs for M2 filtering

### PR DESCRIPTION
**Pull request by davidbenjamin**
@takutosato can you review this?

### CODE REVIEW

File: docs/mutect/mutect.tex [hunk 1]
```diff
@@ -324,4 +324,50 @@ \section{Proposed tumor in normal estimation tool}
...
+\end{itemize}
+
+Additionally, there are two unadjustable filters. the panel of normals filter removes all alleles at a site belonging to the panel of normals, which is a vcf of blacklisted artifact sites.  It can be disabled by not passing a panel of normals to \code{Mutect2}.  There is also an STR contraction filter which removes variants that are the deletion of a single repeat unit of an STR when this repeat unit contains more than one base.  This filter can be disabled with the argument \code{-XA TandemRepeat} which turns off the \code{TandemRepeat} annotation.
+
+Here for convenience is a table of \code{Mutect2} filters with their corresponding annotations specified by the \code{-A} argument\footnote{Most of these are default annotations and do not need to be invoked explicitly.}, vcf keys for these annotations, and command line arguments controlling filtering thresholds.
+
+\begin{table}[h!]
```
Comment (takutosato) [inline docs/mutect/mutect.tex]: Once we update the variable names we should update the argument column

File: docs/mutect/mutect.tex [hunk 2]
```diff
@@ -324,4 +324,50 @@ \section{Proposed tumor in normal estimation tool}
...
+
+\begin{itemize}
+\item  \code{tumor\_lod} is the minimum likelihood of an allele as determined by the somatic likelihoods model required to pass.
+\item \code{maxEventsInHaplotype} is the maximum allowable number of called variants co-occurring in a single assembly region.  If the number of called variants exceeds this they will all be filtered.  Note that this filter is misnamed because it counts the total number of events over all haplotypes in an assembly region.
+\item \code{uniqueAltReadCount} is the minimum number of unique (start position, fragment length) pairs required to make a call.  This count is a proxy for the number of unique molecules (as opposed to PCR duplicates) supporting an allele.  Normally PCR duplicates are marked and filtered by the GATK engine, but in UMI-aware calling this may not be the case, hence the need for this filter.
+\item \code{maxAltAllelesThreshold} is the maximum allowable number of alt alleles at a site.  By default only biallelic variants pass the filter.
+\item \code{max\_germline\_posterior} is the maximum posterior probability, as determined by the above germline probability model, that a variant is a germline event.
```
[the remark this answers is not in the document]
Comment (davidbenjamin) [reply docs/mutect/mutect.tex]: I agree, but there's discussion about standardizing all GATK argument to eg `strand-artifact-posterior-prob-threshold` and I'm waiting to see about that.  Personally I think camel case is superior.  If HaplotypeCaller doesn't adopt a standard within a few weeks, then let's do camel case.

File: docs/mutect/mutect.tex [hunk 3]
```diff
@@ -324,4 +324,50 @@ \section{Proposed tumor in normal estimation tool}
 {\sum_{\rm somatic~SNV~sites} ({\rm depth~in~normal}) \times ({\rm alt~fraction~in~tumor})}
 \end{align}
 
+\section{Mutect filters}
+\code{Mutect2} emits candidate variants with a set of annotations.  After that \code{FilterMutectCalls} produces filtered calls by subjecting these variants to a series of hard filters that reject sites if some annotation is out of an allowable range.  Here are the command line arguments that control these filters, along with the annotations they relate to.
+
+\begin{itemize}
+\item  \code{tumor\_lod} is the minimum likelihood of an allele as determined by the somatic likelihoods model required to pass.
+\item \code{maxEventsInHaplotype} is the maximum allowable number of called variants co-occurring in a single assembly region.  If the number of called variants exceeds this they will all be filtered.  Note that this filter is misnamed because it counts the total number of events over all haplotypes in an assembly region.
```
[the remark this answers is not in the document]
Comment (davidbenjamin) [reply docs/mutect/mutect.tex]: Yeah, I think I'll do it after the comms team's upcoming tutorial.

File: docs/mutect/mutect.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: sounds good

**Pull request merged**